packages/sakuli-forwarder-icinga2/src/icinga2-forwarder.class.ts

```diff
@@ -21,34 +21,24 @@ export class Icinga2Forwarder implements Forwarder {
         await validateProps(this.properties);
         }
         this.logger = logger;
-        ifPresent(this.properties, (props) => this.logDebug(renderIcinga2Properties(props)));
-    }
-
-    logDebug(message: string, ...data: any[]) {
-        ifPresent(this.logger, log => log.debug(message, ...data));
-    }
-    logInfo(message: string, ...data: any[]) {
-        ifPresent(this.logger, log => log.info(message, ...data));
-    }
-    logError(message: string, ...data: any[]) {
-        ifPresent(this.logger, log => log.error(message, ...data));
+        ifPresent(this.properties, (props) => this.logger?.debug(renderIcinga2Properties(props)));
```

Comment:
```suggestion
        ifPresent(this.properties, (props) => this.logger.debug(renderIcinga2Properties(props)));
```
Elvis is not required here as the logger is passed to the setup method. There is no reference of a potential undefined object.

Reply:
In this case, `properties` are passed through and not `logger`